scripts/src/main/resources/scripts/command/docker

```diff
@@ -118,6 +122,26 @@ function doRun() {
   docker "${@}"
 }
 
+function doIsWsl1Used() {
+{
+  echo '@echo off'
+  echo 'ver > tmp-kernel-versiontest-result.txt'
+} > tmp-kernel-versiontest.bat
+
+cmd "/C tmp-kernel-versiontest.bat"
+
+groupVersion=$(cat tmp-kernel-versiontest-result.txt | sed 's/^.*[^0-9][0-9]*\.[0-9][0-9]*\.\([0-9][0-9]*\)\.[0-9][0-9]*.*$/\1/')
+
+rm tmp-kernel-versiontest.bat
+rm tmp-kernel-versiontest-result.txt
+
+if [ "${groupVersion}" -lt 18917 ]; then
```

Comment:
IMHO we can not determine from the windows version what the `wsl` version actually is.
When I have WSL1 and do `wsl --update` I should have WSL2 then but this does not change my windows version, or does it?